server/py/services/api/main.py

```diff
@@ -547,21 +547,27 @@ def _start_periodic_partition_management(self):
                 f"Starting periodic partition management for table {table_name}",
                 retention_days=retention_days,
             )
-            partition_interval = framework.db.session.run_function_with_new_db_session(
-                services.api.utils.db.partitioner.MySQLPartitioner().get_partition_interval,
-                table_name=table_name,
-            )
-            interval_in_seconds = int(
-                partition_interval.as_duration().total_seconds() / 2
-            )
-            run_function_periodically(
-                interval_in_seconds,
-                f"{self._manage_partitions.__name__}_{table_name}",
-                False,
-                self._manage_partitions,
-                table_name=table_name,
-                retention_days=retention_days,
-            )
+            try:
+                partition_interval = framework.db.session.run_function_with_new_db_session(
+                    services.api.utils.db.partitioner.MySQLPartitioner().get_partition_interval,
+                    table_name=table_name,
+                )
+                interval_in_seconds = int(
+                    partition_interval.as_duration().total_seconds() / 2
+                )
+                run_function_periodically(
+                    interval_in_seconds,
+                    f"{self._manage_partitions.__name__}_{table_name}",
+                    False,
+                    self._manage_partitions,
+                    table_name=table_name,
+                    retention_days=retention_days,
+                )
+            except mlrun.errors.MLRunValueError:
+                self._logger.warning(
+                    "Failed to start retention management for table",
+                    table_name=table_name,
+                )
```

Comment:
What does it mean that there is no partition interval? Doesn't it mean that something is wrong?

Reply:
@TomerShor it does mean, but if we raise error here, then chief will exit which sounds like not the best idea

Reply:
But if we don't fail here:
- Something else will fail after MLRun starts
- We won't manage the partitions at all which is also not a good idea.

Can you say what can cause the partition interval to be missing (other than between RCs)?
Also, perhaps instead of ignoring we can use a default partition interval

Reply:
@TomerShor
> Can you say what can cause the partition interval to be missing (other than between RCs)?
 if table is not there. For instance, because of the wrong configuration of mlrun. Or since we define partitioned tables in the config, I can imagine a situation where config from newer mlrun version was taken and imagine we have not only alert_activations, but some other table (`table_x` ) in retention_objects. But mlrun version is older and don't have this table yet. Chief will exit with error without this fix.

Reply:
In that case I can argue the same thing regarding wrong configuration [here](https://github.com/mlrun/mlrun/blob/da480edda6b9a8a09e7211260b1f86e673a95038/mlrun/config.py#L1081-L1090).
Why would we fail in such cases and in other not?

I think its best if we can differentiate between an actual error and a misconfiguration.
I think the case where a table exists, its set in the `object_partitions` but it doesn't have a partition expression is an error we should not necessarily ignore 
If the table doesn't exist then maybe it's ok to ignore.

I understand that ignoring the exception fixes the bug, but we need to be understand what does it mean when we ignore it.

Reply:
@TomerShor the problem in this specific case is that table doesn't exist, so we can't do any retention/partitioning on it.

Reply:
@TomerShor actually, maybe it would be better to raise an error here and make user change mlrun config (remove the wrong table) instead of ignoring it. wdyt?

Reply:
I think that we should raise the error. An informative one though so it's clear what happened